portal-frontend/src/app/services/search/search.dto.ts

```diff
@@ -23,6 +23,7 @@ export interface SearchResponseDto {
   totalApplications: number;
   totalNoticeOfIntents: number;
   totalNotifications: number;
+  outcome: string;
```

Comment:
I don't think the main search result returns an outcome, did you mean to add this to the BaseSearchResultDto above instead?

Reply:
thats being imported in public search dto compoonent

Reply:
I added a comment above with more detail, but I believe this still need to be removed.